docs: Update Logs Visualizations topic (#112541)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/logs/index.md

<!-- prettier-ignore-start -->

> In the Panel options section of the panel editor pane, set basic options like panel title and description, as well as panel links.

Reviewer: This is covered in the section above under the **Panel options** heading (line 78). It uses a shared content file that renders when you build the docs. 
```suggestion
```

|      Option     |   Description   |
| --------------- | --------------- |
| Time   | Show or hide the time column. This is the timestamp associated with the log line as reported from the data source. |
| Unique labels | Show or hide the unique labels column, which shows only non-common labels. |

> | Wrap lines | Turn line wrapping on or off. |

Reviewer 2: Prettify JSON still exists as an option. It's shown when wrapping is enabled

| Highlighting| Experimental. Use a predefined coloring scheme to highlight relevant parts of the log lines. **The new logs panel implements a predefined set of rules to apply subtle colors to the logs lines, to help with readability and help with identifying important information faster. This is an optional feature that can be disabled in the panel options or in the controls on the right.|
| Enable log details | Toggle the switch on to see an extendable area with log details including labels and detected fields. Each field or label has a stats icon to display ad-hoc statistics in relation to all displayed logs. The default setting is on. |

> | Details panel mode |  Choose to display the log details in a sidebar panel or inline, below the log line.  Depending on the viewport size, the default mode it uses can be inline (small viewports) or sidebar (larger viewports). Independent of this default mode, you can also change mode dynamically in the panel by clicking the mode control. |

Reviewer: ```suggestion
| Log details panel mode |  Choose to display the log details in a sidebar panel or inline, below the log line.  The default mode depends on viewport size: the default mode for smaller viewports is inline, while for larger ones, it's sidebar. |
```
